cmd/tailscale/cli/configure-kube.go

```diff
@@ -140,9 +155,23 @@ func updateKubeconfig(cfgYaml []byte, fqdn string) ([]byte, error) {
 }
 
 func setKubeconfigForPeer(fqdn, filePath string) error {
+	dir := filepath.Dir(filePath)
+	if _, err := os.Stat(dir); err != nil {
+		if !os.IsNotExist(err) {
+			return err
+		}
+		if version.IsSandboxedMacOS() {
+			// macOS sandboxing prevents us from creating the .kube directory
```

Comment:
We could add the entire `.kube` directory to the entitlement to allow us to create the directory?

Reply:
Changed it to always attempt creating it and checked explicitly for `os.ErrPermission` before asking user to manually create it.